test/import_test.js

```diff
@@ -13,7 +13,19 @@ function addTest(name, translatorID, input, expected) {
 
 		assert.equal(response.headers['zotero-translator-id'], translatorID);
 
-		assert.deepEqual(response.body, expected);
+		for (const items of [expected, response.body]) {
+			for (const item of items) {
+				delete item.key
+				delete item.version
+				delete item.parentItem
+				for (const array of ['tags', 'notes', 'attachments', 'seeAlso']) {
+					if (Array.isArray(item[array]) && !item[array].length) delete item[array];
```

Comment:
Braces

(Conditionals should almost always have braces in Zotero code. We don't require them in the ESLint config just because it's occasionally useful to omit them — an immediate `return;` at the top of a function, say — but that's by far the exception.)

Reply:
I think there's just a difference of preference here - I generally prefer const unless let is required, so that's what will naturally show up in code as I'm writing it. It's not a matter of deep principle for me, should I change this to let?